FormalConjectures/ErdosProblems/312.lean

```diff
@@ -0,0 +1,42 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+import FormalConjectures.Util.Answer.Syntax
+
+namespace FormalConjectures
+namespace ErdosProblems
+namespace Erdos312
+
+/--
+Does there exist a constant `c > 0` such that for every `K > 1`, whenever the total
+harmonic sum `∑ i, 1 / a i` exceeds `K` and `n` is large enough, we can choose a subset
+of indices whose harmonic sum lies in `(1 - exp (-(c*K)), 1]`?
```

Comment:
```suggestion
Does there exist a constant $c > 0$ such that, for any $K > 1$, whenever $A$ is a sufficiently large finite multiset of integers with $\sum_{n\in A}\frac{1}{n}>K$ there exists some $S \subset A$ such that 
$$1-e^{-cK} < \sum_{n\in S}\frac{1}{n}\leq 1?$$
```
Perhaps let's stick to the original formulation, I think using `a` and `S` makes it sufficiently clear how the multiset in the informal statement is modeled here.